src/openfl/display/Bitmap.hx

```diff
@@ -234,6 +238,37 @@ class Bitmap extends DisplayObject implements IShaderDrawable {
 	}
 	
 	
+	function __getBatchQuad (renderSession:RenderSession):Quad {
+		
+		if (__batchQuadDirty) {
+			if (__batchQuad == null) {
+				__batchQuad = new Quad ();
+			}
+			
+			var snapToPixel = renderSession.roundPixels || __snapToPixel ();
+			var transform = (cast renderSession.renderer : GLRenderer).getDisplayTransformTempMatrix (__renderTransform, snapToPixel);
```

Comment:
Hmm, I wonder if we should pass `GLRenderer` here and access `renderSession` from its field instead...